com.civicrm.hrjobroles/views/modalDialog.html

```diff
@@ -1,11 +1,18 @@
 <div class="modal-header">
-    <button type="button" class="close" ng-click="cancel()"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
-    <h4 class="modal-title">{{title}}</h4>
+  <button type="button" class="close" ng-click="cancel()">
+    <span aria-hidden="true">&times;</span>
```

Comment:
`&times;` is included in the default [Bootstrap markup](http://getbootstrap.com/javascript/#modals) for modals. We use it even for L&A